app/src/main/res/values-de/strings.xml

```diff
@@ -115,6 +115,8 @@
     <string name="enable_sms">Eingehende Textnachrichten im Verlauf anzeigen</string>
     <string name="enable_phone">Eingehende Anrufe im Verlauf anzeigen</string>
     <string name="enable_app">Neu installierte Apps im Verlauf anzeigen</string>
+    <string name="enable_favorites_bar">Favoriten über der Suche anzeigen</string>
```

Comment:
Nur eine Kleinigkeit, aber der "offizielle" Begriff ist in diesem Fall *Suchfeld* nicht *Suche* (s. `main_clear`)